src/data/faq.json

```diff
@@ -376,6 +379,217 @@
+                                "anchor": "warn_without_tan_still_positive",
+                                "active": false,
+                                "textblock": [
+                                    "We further recommend that after a positive self-test, you behave reasonably on your own responsibility in a way that does not infect others. If you behave in this way, no further warning is necessary.",
+                                    "To prevent the feature from being abused and creating unnecessary warnings that are not based on real risk contacts, it was decided that a user with the new feature can warn others only once within a certain period of time.",
+                                    "After this period has elapsed, it is then possible again to issue a warning without a TAN based on your own positive test result. This restriction does not apply to positive test results transmitted via app."
+                                ]
+                            },
+                            {
+                                "title": "Do I need to take a picture of my test to prove the test?",
+                                "anchor": "warn_without_tan_photo",
+                                "active": false,
+                                "textblock": [
+                                    "No, the Corona-Warn-App does not provide for uploading photos or other documents to prove that a test really took place, or to process them in any other way. We deliberately rely on the prudence and personal responsibility of our users."
+                                ]
+                            },
+                            {
+                                "title": "How often can I warn without TAN?",
+                                "anchor": "warn_without_tan_how_often",
+                                "active": false,
+                                "textblock": [
+                                    "Within a certain period you can warn once without TAN. After the blocking period has expired, you can warn again without a TAN. In principle, there are no upper limits to how often you can warn without a TAN, as long as the intervals are not undercut."
+                                ]
+                            },
+                            {
+                                "title": "Can I warn on behalf of family members, such as my infected child?",
+                                "anchor": "warn_without_tan_family",
+                                "active": false,
+                                "textblock": [
+                                    "No, this is not technically possible because the risk contacts of the person tested positive were not necessarily also recorded on their smartphone by contact tracing. Warning for others from your smartphone would therefore not reach these contacts at all, but only your (i.e. the wrong) contacts."

```

Comment:
"on their smartphone"
=> 
"on your smartphone"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/f2b4444eace8ae8143537742d5f8e4f971a82b60, thanks!